client/app/dedicatedCloud/user/edit/dedicatedCloud-user-edit.html

```diff
@@ -0,0 +1,112 @@
+<form name="userEditForm"
+      data-ng-submit="$ctrl.editUser()"
+      novalidate>
+
+    <oui-modal data-title="{{:: 'dedicatedCloud_USER_modify' | translate }}"
+               data-primary-label="{{ 'wizard_confirm' | translate }}"
+               data-secondary-label="{{ 'wizard_close' | translate }}"
+               data-secondary-action="$ctrl.$state.go('^')"
+               data-on-dismiss="$ctrl.$state.go('^')"
+               data-loading="$ctrl.loading">
+
+        <label>
```

Comment:
Out of curiosity, is there any reason why you have your input inside you label instead of below ?

`<label data-translate="dedicatedCloud_USER_firstname"></label>
<input type="text"
       class="form-control"
       data-ng-model="$ctrl.user.firstName">`

Not that I mind 😄